### PR TITLE
add works section

### PR DESCRIPTION
**Pull request by mj-e-boc**
-Add works Section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
         <ul class="navbar">
             <li>
                 <a href="#">
-                    <h1 class="logo">M Jay</h1>
+                    <h2 class="logo">M Jay</h2>
                 </a>
             </li>
             <li class="bars"><i class="fa-solid fa-bars"></i></li>
```
Comment (1GORDON) [inline index.html:23]: Great work @mj-e-boc🤝

- [ ] [Optional] Try to make the hamburger icon a clickable link. You may surround it with an `<a>` tag

File: styles.css [hunk 2]
```diff
@@ -1,107 +1,188 @@
...
+  margin-right: 15%;
+}
+
+.button {
+  background-color: #7d73fc;
+  color: white;
+  height: 48px;
+  padding: 12px;
+  font-family: "Roboto Condensed", sans-serif;
+}
```
Comment (1GORDON) [inline styles.css:188]: Great work @mj-e-boc 🤝

- [ ] As pointed out in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md). You may please implement the button interactions hover and enable.

File: styles.css [hunk 3]
```diff
@@ -1,107 +1,188 @@
...
+  font-style: normal;
+  font-weight: 100;
+  font-size: 20px;
+  line-height: 28px;
+  color: #c8c8d4;
+  text-align: center;
+  border-bottom: 1px solid #646369;
+  padding-bottom: 33px;
+  font-family: "Roboto Condensed", sans-serif;
 }
```
Comment (1GORDON) [inline styles.css:72]: Nice design @mj-e-boc 🤝

- [ ] You may please try to use the right font-family as it is provided in the Figma template.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @mj-e-boc ✋

Good job so far!

## Highlights!
- Followed Git flow ✔️ 
- Good commit messages  ✔️ 
- Professional Readme ✔️
- Professional Pull request ✔️
- Great use of semantic tags ✔️
- Project is working as expected ✔️
- Linters are passing ✔️    

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -66,6 +66,73 @@ <h2 class="title">Software engineer</h2>
             <li><img src="/Images/Vector.png" alt="" /></li>
         </ul>
     </div>
+
+    <div class="bod">
```
[the remark this answers is not in the document]
Comment (rloterh) [reply index.html]: This issue is well resolved! ✔️

File: index.html [hunk 5]
```diff
@@ -66,6 +66,73 @@ <h2 class="title">Software engineer</h2>
...
+        <footer class="foot">
+            <ul class="list">
+                <li class="itembod">css</li>
+                <li class="itembod">html</li>
+                <li class="itembod">bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+        </footer>
+        <div class="buttonCont">
+            <button class="button" type="button">see Project</button>
```
[the remark this answers is not in the document]
Comment (rloterh) [reply index.html:98]: This issue is well resolved! ✔️

File: index.html (hunk 1 above)
Comment (rloterh) [reply index.html:23]: This issue is well resolved! ✔️

File: styles.css (hunk 2 above)
Comment (rloterh) [reply styles.css]: This issue is well resolved! ✔️

File: styles.css (hunk 3 above)
Comment (rloterh) [reply styles.css:72]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi ,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Highlights 👍
- App matches design template


<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GedeonTS — COMMENTED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @mj-e-boc,

You are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- No linter errors :clamp: 
- Good use of GitHub flow
## Optional suggestions :small_airplane: 
- [OPTIONAL] I think it would be better to write a more descriptive PR summary, as it would ma ke your project look more professional

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**